ingesters/SimpleRelay/main.go

```diff
@@ -88,6 +89,14 @@ func mainInit() {
 }
 
 func main() {
+	f, err := os.Create("/tmp/cpu.prof")
```

Comment:
We probably don't want profiling enabled on every run

Reply:
great catch.